rubocop.yml

```diff
@@ -656,7 +656,7 @@ Style/TrailingCommaInArrayLiteral:
   EnforcedStyleForMultiline: comma
 
 Style/TrailingCommaInHashLiteral:
-  EnforcedStyleForMultiline: comma
+  Enabled: false
```

Comment:
Here I disagree. The trailing commas have a huge advantage: If you later add more lines to the hash the Diff in the MR/PR just highlights the one line you actually changed. And the respective commit message is also only for the line you changed. So I would really like to enforce this.